game_eggs/minecraft/crossplay/purpur-geysermc-floodgate/README.md

```diff
@@ -11,9 +11,9 @@ Purpur-GeyserMC-Floodgate is a drop-in replacement for Paper servers designed fo
 
 ## Server Ports
 
-The minecraft server requires a single port for access (default 25565). GeyserMC requires an additional port (default 19132). Any other plugins you add may require extra ports to enabled for the server.
+The minecraft server requires a single port for access (default 25565 TCP). GeyserMC requires an additional port (default 19132 UDP). Any other plugins you add may require extra ports to enabled for the server.
```

Comment:
No need to add the protocol here. In the table fine not here.

Reply:
Fixed.